### DISCUSSION TITLE
请问如何让newbing查看本地文件？比如ppt之类的

### DISCUSSION THREAD

**Discussion by falsemaskx** [viopsa233/AI-testers-Forum#63] [Q&A]
想试试用侧边栏来分析本地文件，但是edge一打开ppt就跳转了，根本无法打开

**Comment by viopsa233** [comment]
PDF试过没有

**Comment by q2408808** [comment]
你可以看看这个：https://www.youtube.com/watch?v=A3F5riM5BNE ，对应的github是 https://github.com/imartinez/privateGPT

**Comment by juzeon** [comment]
[SydneyQt](https://github.com/juzeon/SydneyQt) 有读取本地文档（doc、pdf、ppt）的功能。

**Reply by kamanqi** [reply]
好用
